src/screens/VisualTests/SnapshotControls.tsx

```diff
@@ -116,68 +112,65 @@ export const SnapshotControls = ({
 
   return (
     <>
-      <Label>
-        <Text>
-          <b>
-            {baselineImageVisible ? "Baseline" : "Latest"}
-            <span> snapshot</span>
-          </b>
-        </Text>
-      </Label>
+      {hasControls && (
```

Comment:
Do we cover this in a story anywhere?

Reply:
Yeah all the stories where there is no baseline, there are no controls (and no latest/baseline label).